server/chat-commands/core.ts

```diff
@@ -1453,18 +1451,15 @@ export const commands: ChatCommands = {
 				title: targetRoom.title,
 				type: targetRoom.type,
 				visibility: visibility,
-				modchat: targetRoom.modchat,
-				modjoin: targetRoom.modjoin,
+				modchat: targetRoom.settings.modchat,
+				modjoin: targetRoom.settings.modjoin,
 				auth: {},
 				users: [],
 			};
 
-			if (targetRoom.auth) {
-				for (const userid in targetRoom.auth) {
-					const rank = targetRoom.auth[userid];
-					if (!roominfo.auth[rank]) roominfo.auth[rank] = [];
-					roominfo.auth[rank].push(userid);
-				}
+			for (const [k, v] of targetRoom.auth) {
```

Comment:
`k, v` should not be used because it's unclear what they mean; I'd use `userid, groupSymbol` instead, here and everywhere else `auth` is iterated.